plugins/BEdita/API/src/Controller/UsersController.php

```diff
@@ -22,10 +22,8 @@
  * Controller for `/users` endpoint.
```

Comment:
Unused `use` statements - not part of this PR, but since class logic is inherited they are not needed